Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -118,11 +147,24 @@ class ContentBlockerHelper {
         self.tab = tab
         self.profile = profile
 
+        if AppConstants.IsRunningTest {
+            ContentBlockerHelper.testInstance = self
```

Comment:
Is there a less-intrusive way of doing this? Although, I'm guessing the compiler probably optimizes this to a `NOOP`

Reply:
This is the simplest way I know of to expose an instance to the test framework, but it is a runtime check that isn't compiled out. 
Preprocessor flags would work for us because we have different build schemes for test, thus it is easy to have a `-D UNDER_TEST` and have `#if UNDER_TEST`. I'll mention in next eng meeting it might be useful for future to do this.